Source/UI/IdentityUIConfiguration.swift

```diff
@@ -19,6 +19,17 @@ public struct IdentityUIConfiguration {
     public let tracker: TrackingEventsHandler?
     /// This determines whether you want to allow the user to dismiss the login flow
     public let isCancelable: Bool
+    /// This will determin if you want to use biometric login.
+    public var useBiometrics: Bool {
+        get {
+            // This will return false if the key is not present.
+            let value = UserDefaults.standard.bool(forKey: "Identity.useBiometrics")
```

Comment:
Same this about using the `Settings` object.

Reply:
I can't use settings for defaults here. So I think I will skip this.

Reply:
So I see it's because of the use of it in the default argument. Suggestion to not use `UserDefaults` as the default value. You can do this:
```
@@ -74,7 +73,7 @@ public struct IdentityUIConfiguration {
         theme: IdentityUITheme = .default,
         isCancelable: Bool = true,
         isSkippable: Bool = false,
-        useBiometrics: Bool = UserDefaults.standard.bool(forKey: "Identity.useBiometrics"),
+        useBiometrics: Bool? = nil,
         presentationHook: ((UIViewController) -> Void)? = nil,
         tracker: TrackingEventsHandler? = nil,
         localizationBundle: Bundle? = nil,
@@ -90,7 +89,7 @@ public struct IdentityUIConfiguration {
         if let appName = appName {
             self.appName = appName
         }
-        self.useBiometrics = useBiometrics
+        self.useBiometrics = useBiometrics ?? self.useBiometrics
     }
```

Now you can use `Settings` inside the `useBiometrics` property.